packages/utilities/psammead-locales/moment/pcm.js

```diff
@@ -0,0 +1,60 @@
+const moment = require('moment');
+
+moment.defineLocale('pcm', {
+  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split(
+    '_',
+  ),
+  monthsShort: 'Jan_Feb_Mar_Apr_May_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
+  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split(
+    '_',
+  ),
+  weekdaysShort: 'Sun_Mon_Tue_Wed_Thu_Fri_Sat'.split('_'),
+  weekdaysMin: 'Su_Mo_Tu_We_Th_Fr_Sa'.split('_'),
+  longDateFormat: {
+    LT: 'HH:mm',
+    LTS: 'HH:mm:ss',
+    L: 'DD/MM/YYYY',
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, D MMMM YYYY HH:mm',
```

Comment:
Translations doc makes it look like this should be the same as LLL?

Reply:
Which line? Can you copy the example please :P

Reply:
LLL: 3rd March 2019 14:25
LLLL: 3rd March 2019 14:25
lines 97 & 98

Reply:
Ahh good spot, done :)